frontend/src/lib/lemon-ui/Sparkline.tsx

```diff
@@ -111,9 +111,11 @@ export function Sparkline({
                                     lineHeight: 1,
                                 },
                             },
+                            border: {
+                                dash: [2],
+                                display: false,
+                            },
                             grid: {
-                                borderDash: [2],
-                                drawBorder: false,
                                 display: true,
                                 tickLength: 0,
                             },
```

Comment:
**note:** The addition of border here is introduced because `grid.drawBorder` and `grid.borderDash` have been renamed to `border.display` and `border.dash` in Chart.js v4.

Additional context from the [migration guide](https://www.chartjs.org/docs/latest/migration/v4-migration.html):
- scales[id].grid.drawBorder has been renamed to scales[id].border.display
- scales[id].grid.borderDash has been renamed to scales[id].border.dash